CONTRIBUTING.md

```diff
@@ -11,6 +11,23 @@ Bloom is a tool which will help you do the release as well as open a pull-reques
 It will also assist adding documentation and source entries.
 There are [several helpful tutorials](http://wiki.ros.org/bloom/Tutorials) which provide instructions on how to do things like [make a first release](http://wiki.ros.org/bloom/Tutorials/FirstTimeRelease). 
 
+### Guidelines for Package Naming
+
+When releasing a new package into a distribution, please follow the naming guidelines set out in the [ROS REP 144: ROS Package Naming](https://www.ros.org/reps/rep-0144.html).
+
+### Binary Release Follow-up
+
+Once a `ros/rosdisto` pull request is merged, the package will be built in the ROS build farm.
+It is important that the author follow up to verify that the package has successfully built for a particular distribution.
+This can be checked via both the distribution status page:
+* For a ROS Distribution, use `repositories.ros.org/status_page/`, for example [ROS packages for Melodic](http://repositories.ros.org/status_page/ros_melodic_default.html).
+* For a ROS 2 Distribution, use `repo.ros2.org/status_page/`, for example [ROS packages for Foxy](http://repo.ros2.org/status_page/ros_foxy_default.html).
+
+Additionally, it may be necessary to determine build failures, these jobs are located at:
+* [ROS 2 Buildfarm](http://build.ros2.org/)
+* [ROS Buildfarm](http://build.ros.org/)
+
+If a package continuously fails to build, the ROS Boss for that distribution may choose to revert the pull request that introduced it.
```

Comment:
```suggestion
If a package continuously fails to build, the ROS Boss for that distribution may choose to revert the pull request that introduced it.
Or they may take [other actions](https://github.com/ros-infrastructure/ros_buildfarm/blob/master/doc/ongoing_operations.rst) to avoid repeated failures.
```